server/test/controllers/users.e2e-spec.ts

```diff
@@ -35,45 +37,8 @@ describe('users controller', () => {
     });
   });
 
-  describe('change password', () => {
-    it('should allow users to update their password', async () => {
-      const userData = await createUser(app, { email: 'admin@tooljet.io', role: 'admin' });
-      const { user, orgUser } = userData;
-
-      const oldPassword = user.password;
-
-      const response = await request(app.getHttpServer())
-        .patch('/api/users/change_password')
-        .set('Authorization', authHeaderForUser(user))
-        .send({ currentPassword: 'password', newPassword: 'new password' });
-
-      expect(response.statusCode).toBe(200);
-
-      await user.reload();
-
-      expect(user.password).not.toEqual(oldPassword);
-    });
-
-    it('should not allow users to update their password if entered current password is wrong', async () => {
-      const userData = await createUser(app, { email: 'admin@tooljet.io', role: 'admin' });
-      const { user, orgUser } = userData;
-
-      const oldPassword = user.password;
-
-      const response = await request(app.getHttpServer())
-        .patch('/api/users/change_password')
-        .set('Authorization', authHeaderForUser(user))
-        .send({ currentPassword: 'wrong password', newPassword: 'new password' });
-
-      expect(response.statusCode).toBe(403);
-
-      await user.reload();
-
-      expect(user.password).toEqual(oldPassword);
-    });
-  });
-
   afterAll(async () => {
+    await getConnection().close();
```

Comment:
Had to specify this to get the suite working, without this change app fails with [test suite](https://github.com/ToolJet/ToolJet/runs/4462789088?check_suite_focus=true#step:9:65) not being able to run.
But now with this change, I get error on the suite [suite](https://github.com/ToolJet/ToolJet/runs/4466987549?check_suite_focus=true#step:9:72).